openmetadata-ui/src/main/resources/ui/src/components/Chart/DataDistributionHistogram.component.tsx

```diff
@@ -112,7 +112,9 @@ const DataDistributionHistogram = ({
                     />
                     <Legend />
                     <Tooltip
-                      formatter={(value: number) => tooltipFormatter(value)}
+                      formatter={(value: number | string) =>
+                        tooltipFormatter(value)
+                      }
```

Comment:
```suggestion
                      formatter={tooltipFormatter}
```

Reply:
It won't work as `tooltipFormatter` accepts one more optional parameter which is `tickFormatter` and the `formatter` function also sends different values at 2, 3 position.